### PR TITLE
sets funding_status_id to user selection or tentative if no selection

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/8721

## Testing
**URL to test:** 
https://deploy-preview-630--atd-moped-main.netlify.app/

**Steps to test:**
1) create a new funding source setting a status value
2) create a new funding source without setting a status value

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

**Comment by tillyw** [comment]
in addressing this issue, had a conversation with @amenity about setting value to null (displayed as blank value in ui) if user does not select a value but decided to set value to 1 ("tentative") in these cases. setting value as null creates conflicts with the way projects are archived (or soft deleted) because they are not displayed in the ui if they have a null value. graphql query for displaying records only displayed records with value not equal to 0, which in this case appears to include null values. @frankhereford suggested changing the way we archive or soft delete projects by altering schema to include an "archived" boolean or similar stragegy.

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨ 

betty-white-dab

File: moped-editor/src/views/projects/projectView/ProjectFundingTable.js [hunk 1]
```diff
@@ -676,7 +676,7 @@ const ProjectFundingTable = () => {
               }),
           onRowUpdate: (newData, oldData) => {
             const updateProjectFundingData = newData;
-
+
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectFundingTable.js:679]: Do you mind running this through prettier? 🙏

File: moped-editor/src/views/projects/projectView/ProjectFundingTable.js [hunk 2]
```diff
@@ -657,7 +657,7 @@ const ProjectFundingTable = () => {
                   ...newData,
                   project_id: projectId,
                   added_by: getDatabaseId(user),
-                  funding_status_id: 1,
+                  funding_status_id: newData.funding_status_id || 1,
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectFundingTable.js:660]: This works great with the unspecified default being tentative or user input if provided. 🙌

**Review by frankhereford — APPROVED** [review]
🎆 🙏 🚢

**Comment by amenity** [comment]
Nice! My only potential tweak would be if we could show "Tentative" already populated when they add a new line. So they aren't surprised when they submit and it's filled in:

**Comment by tillyw** [comment]
@amenity updated pr so that dropdown is automatically set to tentative unless user selects another option. please take a look!

**Review by mateoclarke — APPROVED** [review]
👍

**Review by amenity — APPROVED** [review]
LGTM! Thanks, Tilly!

**Pull request merged**